glossary.rst

```diff
@@ -84,5 +62,36 @@ Glossary
         .. note::
             If :term:`analytics attribute` uses the same name - it will be represented as a separate :term:`attribute`.
 
-    Analytics
-        PPAS component gathering statistics about each :term:`user` of the :term:`application` (previously **Piwik**).
+    Device ID
+        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
+        device. Device IDs are separate from hardware serial numbers.
+
+    Identifier
+        Unique :term:`user` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).
+
+    JavaScript Tracking Client (JSTC)
+        A JavaScript object that is able to send requests to CPP. It is loaded and created with download of `ppms.js` file. It has an :ref:`API<data-collection-javascript-tracking-client-api>` that allows to configure what data requests should contain. You can :ref:`learn more about JSTC here<data-collection-javascript-tracking-client-installation>`
+
+    JavaScript Tracking Snippet (JSTS)
+        A JavaScript code, usually in form of HTML tag, that initiates JSTC and sends first tracking request. You can see an :ref:`example of JSTS here<jtc-installation-jsts-example>`.
+
+    PII
+        Personally Identifiable Information.
+
+    Tracking Tag
+        A HTML tag, that is created and configured by Tag Manager. It is loaded to the website with Tag Manager Container. Using a Tracking Tag is an alternative for implementing a JavaScript Tracking Snippet. You can :ref:`learn more about Tracking Tag here<https://help.piwik.pro/support/tag-manager/piwik-pro-tag/>`.
+
+    User
+        Visitor on tracked :term:`application`.
+
+    User ID
+        Permanent ID assigned to :term:`user` by :term:`application` (e.g. username). You can read more about it
+        `here <https://help.piwik.pro/tag-manager/userid/>`_.
+
+    Visit
+        Period of continuous :term:`user` activity on :term:`application`. It ends in the following situations:
+
+            - after a period of inactivity (option set to 30 minutes by default)
+            - at midnight (option enabled by default)
```

Comment:
This is not true any more. Sessions are not split if midnight occurs in new tracker (new tracker is not aware of tracked app's time zone).

Reply:
Removed